spec/features/users/new_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe 'new user', type: :feature do
+  describe 'When user visits "/register"' do
+    before(:each) do
+      @user_1 = User.create!(name: 'Sam', email: 'sam_t@email.com')
+
+      visit register_user_path
```

Comment:
Smart adding the line to `visit register_user_path` in the before block since we will be starting there for all of the tests in this spec.